internal/commands/attest/attest.go

```diff
@@ -46,7 +46,7 @@ type options struct {
 func (o *options) AddFlags(cmd *cobra.Command) {
 	cmd.Flags().StringVar(&o.FlagObjectType, "objtype", FlagObjectTypeCommit, "[commit | tree] - Git object type to attest")
 	cmd.Flags().StringVarP(&o.FlagPath, "filepath", "f", "", "attestation filepath")
-	cmd.Flags().StringVar(&o.FlagAttestationType, "type", "", `specify a predicate type (slsaprovenance|link|spdx|spdxjson|cyclonedx|vuln|custom) or an URI (default "custom")`)
```

Comment:
After responding to https://github.com/sigstore/gitsign/issues/623#issuecomment-2608287770 this made me realize there was a good reason to have this, which was to support making it easy to add attestations for arbitrary files that don't fit neatly into an in-toto type, e.g. being able to do something like `gitsign attest -f test-results.xml --type custom` and have that work auto-magically.

That said, I think there's more benefits to getting this working with the new in-toto libraries at the moment so happy to proceed with this for now, but this will likely be something we'll look to add back in some form (either as this flag or another flag/command).